### PR TITLE
Dialog tests in IE8 with jQuery 1.7

### PR DESCRIPTION
**Pull request by scottgonzalez**
This gets us to green for dialog in IE8 with jQuery 1.7.

### CODE REVIEW

File: tests/unit/dialog/dialog_core.js [hunk 1]
```diff
@@ -59,90 +59,94 @@ asyncTest( "focus tabbable", function() {
 
 		element = $( markup ).dialog( options );
 		setTimeout(function() {
-			testFn();
-			element.remove();
-			setTimeout( next );
+			testFn(function done() {
```
Comment (jzaefferer) [inline tests/unit/dialog/dialog_core.js]: Just wondering: Is the `done` function name necessary? Or just a help for debugging?

**Comment by jzaefferer** [comment]
:+1:

File: tests/unit/dialog/dialog_core.js (hunk 1 above)
Comment (scottgonzalez) [inline tests/unit/dialog/dialog_core.js]: Just for debugging.

Comment (jzaefferer) [inline tests/unit/dialog/dialog_core.js]: :+1:

**Pull request closed without merging**